db/seeds.rb

```diff
@@ -1,7 +1,54 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require 'json'
+File.open("db/schools_data.json").each do |line|
+  parsed_course = JSON.parse(line)
+  # Every line in a the .txt file represents an JSON course object with
```

Comment:
Unnecessary comment. It is declared on line 3 that JSON is used

Reply:
Removing the comment